src/draw/sw/lv_draw_sw.h

```diff
@@ -35,6 +35,7 @@ typedef struct {
 #if LV_USE_OS
     lv_thread_sync_t sync;
     lv_thread_t thread;
+    volatile bool inited;
```

Comment:
The correct word is "initialized"

Reply:
We are really abut avoiding abbreviation in the API, however "inited" is used in many other places as well. So let's use it now, and review all occurrences separately if required.